bindings/swift/native/make_framework.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -e
+
+while [ $# -gt 0 ]; do
+	if [[ $1 == *"--"* ]]; then
+        v="${1/--/}"
+        declare $v="$2"
+	fi
+	shift
+done
+
+if [ -z "$target" ]; then 
+	echo "--target is missing"
+	exit 1
+fi
+
+FRAMEWORK_NAME=IOTAWalletInternal
+FRAMEWORK_VERSION=1.0.0
+
+makeFramework() {
+	cd $1
+	rm -rf $FRAMEWORK_NAME.framework
+	mkdir $FRAMEWORK_NAME.framework
+	mkdir $FRAMEWORK_NAME.framework/Headers
+	mkdir $FRAMEWORK_NAME.framework/Modules
+
+	touch $FRAMEWORK_NAME.framework/Modules/Modules.modulemap
+	tee -a $FRAMEWORK_NAME.framework/Modules/Modules.modulemap > /dev/null <<EOT
+framework module $FRAMEWORK_NAME {
+  umbrella header "$FRAMEWORK_NAME.h"
+
+  export *
+  module * { export * }
+}
+EOT
+
+	touch $FRAMEWORK_NAME.framework/Info.plist
+	tee -a $FRAMEWORK_NAME.framework/Info.plist > /dev/null <<EOT
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+   <key>CFBundleDevelopmentRegion</key>
+   <string>en</string>
+   <key>CFBundleExecutable</key>
+   <string>${FRAMEWORK_NAME}</string>
+   <key>CFBundleIdentifier</key>
+   <string>org.iotafoundation.${FRAMEWORK_NAME}</string>
+   <key>CFBundleInfoDictionaryVersion</key>
+   <string>6.0</string>
+   <key>CFBundleName</key>
+   <string>${FRAMEWORK_NAME}</string>
+   <key>CFBundlePackageType</key>
+   <string>FMWK</string>
+   <key>CFBundleShortVersionString</key>
+   <string>1.0</string>
+   <key>CFBundleVersion</key>
+   <string>${FRAMEWORK_VERSION}</string>
+</dict>
+</plist>
+EOT
```

Comment:
Same as above, but can't they be individual files checked into the repository? Then we just copy them.

Reply:
It can't. Otherwise I need to copy the content and replace all the variables inside of it manually.
In this way we have more control and no confusion about where the templates can be.